tests/test_dtel.py

```diff
@@ -0,0 +1,289 @@
+from swsscommon import swsscommon
+import time
+import re
+import json
+
+class TestDtel(object):
+    def test_DtelGlobalAttribs(self, dvs):
+    
+        db = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+    
+        # create DTel global attributes in config db
+        tbl = swsscommon.Table(db, "DTEL")
+
+        fvs = swsscommon.FieldValuePairs([("SWITCH_ID", "1")])
+        tbl.set("SWITCH_ID", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("FLOW_STATE_CLEAR_CYCLE", "10")])
+        tbl.set("FLOW_STATE_CLEAR_CYCLE", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("LATENCY_SENSITIVITY", "100")])
+        tbl.set("LATENCY_SENSITIVITY", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("INT_ENDPOINT", "TRUE")])
+        tbl.set("INT_ENDPOINT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("INT_TRANSIT", "TRUE")])
+        tbl.set("INT_TRANSIT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("POSTCARD", "TRUE")])
+        tbl.set("POSTCARD", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("DROP_REPORT", "TRUE")])
+        tbl.set("DROP_REPORT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("QUEUE_REPORT", "TRUE")])
+        tbl.set("QUEUE_REPORT", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("Ethernet0", "Ethernet0"), ("Ethernet4", "Ethernet4")])  
+        tbl.set("SINK_PORT_LIST", fvs)
+
+        fvs = swsscommon.FieldValuePairs([("INT_L4_DSCP_VALUE", "128"), ("INT_L4_DSCP_MASK", "255")])  
+        tbl.set("INT_L4_DSCP", fvs)
+
+        time.sleep(1)
+
+        atbl = swsscommon.Table(adb, "ASIC_STATE:SAI_OBJECT_TYPE_DTEL")
+        keys = atbl.getKeys()
+        
+        for k in keys:
+            (status, fvs) = atbl.get(k)
+            assert status == True
+            
+            for fv in fvs:
+            if fv[0] == "SAI_DTEL_ATTR_SWITCH_ID":
+            assert fv[1] == "1"
```

Comment:
the syntax here looks strange to me, can you check?

Reply:
Converting tabs to spaces messed this up. Will fix it.